Index: contrib/asdf-module.mk
===================================================================
--- contrib/asdf-module.mk
+++ contrib/asdf-module.mk	2012-01-12 09:40:55.347797454 +0100
@@ -9,6 +9,9 @@
 
 UNAME:=$(shell uname -s)
 
+ifeq (Linux,$(UNAME))
+  EXTRA_CFLAGS+=-D_GNU_SOURCE
+endif
 ifeq (SunOS,$(UNAME))
   EXTRA_CFLAGS=-D_XOPEN_SOURCE=500 -D__EXTENSIONS__
   PATH:=/usr/xpg4/bin:${PATH}
