--- ./make-target-contrib.sh.PERMISSIVE	2014-01-21 18:27:50.578413076 +0200
+++ ./make-target-contrib.sh	2014-01-21 18:29:13.943598472 +0200
@@ -69,7 +69,7 @@
 
 for i in $contribs_to_build; do
     test -d contrib/$i && test -f contrib/$i/Makefile || continue;
-    test -f contrib/$i/test-passed && rm contrib/$i/test-passed # remove old convention
+    # test -f contrib/$i/test-passed && rm contrib/$i/test-passed # remove old convention
     test -f obj/asdf-cache/$i/test-passed.test-report && rm obj/asdf-cache/$i/test-passed.test-report
     mkdir -p obj/asdf-cache/$i/
     # hack to get exit codes right.
