Index: make-target-1.sh
===================================================================
--- make-target-1.sh
+++ make-target-1.sh	2012-01-12 09:56:20.976225508 +0100
@@ -20,6 +20,28 @@
 LC_ALL=C
 export LANG LC_ALL
 
+# Allow the definition of INSTALL_ROOT and/or SBCL_HOME to correctly
+# set the hard-coded SBCL_HOME macro in src/runtime/runtime.c
+
+DEFAULT_SBCL_HOME=${DEFAULT_SBCL_HOME:-/usr/local/lib/sbcl/}
+
+# Strip off any trailing / on the name; we'll add this later but don't
+# need two
+DEFAULT_SBCL_HOME=${DEFAULT_SBCL_HOME%/}
+
+export DEFAULT_SBCL_HOME
+
+file=src/runtime/sbcl-home.h
+echo "/* This is a machine-generated file.            */" > $file
+echo "/* Please do not edit it by hand.               */" >> $file
+echo "/* Change the default SBCL_HOME by setting the  */" >> $file
+echo "/* DEFAULT_SBCL_HOME environment variable prior */" >> $file
+echo "/* to building. See make-target-1.sh for more   */" >> $file
+echo "/* information.                                 */" >> $file
+echo "#ifndef SBCL_HOME" >> $file
+echo "#define SBCL_HOME \"$DEFAULT_SBCL_HOME/\"" >> $file
+echo "#endif" >> $file
+
 # Load our build configuration
 . output/build-config
 
Index: src/runtime/runtime.c
===================================================================
--- src/runtime/runtime.c
+++ src/runtime/runtime.c	2012-01-12 09:57:19.241497092 +0100
@@ -68,9 +68,7 @@
 #include "interr.h"
 #endif
 
-#ifndef SBCL_HOME
-#define SBCL_HOME SBCL_PREFIX"/lib/sbcl/"
-#endif
+#include "sbcl-home.h"
 
 #ifdef LISP_FEATURE_HPUX
 extern void *return_from_lisp_stub;
