Index: src/runtime/linux-os.c
===================================================================
--- src/runtime/linux-os.c
+++ src/runtime/linux-os.c	2012-01-12 09:37:31.690343523 +0100
@@ -280,6 +280,10 @@
                 if (i != -1) {
                     runtime[i] = '\0';
                     execve(runtime, argv, envp);
+                } else {
+                    /* if /proc isn't available (like in chroot builds, like mock), 
+                     * try using execvp with argv[0] instead */
+                    execvp(argv[0], argv);
                 }
             }
             /* Either changing the personality or execve() failed. Either
