Index: src/runtime/Config.x86-linux
===================================================================
--- src/runtime/Config.x86-linux
+++ src/runtime/Config.x86-linux	2012-12-01 21:35:12.099411348 +0100
@@ -42,6 +42,8 @@
   OS_LIBS += -lz
 endif
 
+CFLAGS += $(RPM_OPT_FLAGS)
+
 GC_SRC = gencgc.c
 
 # Nothing to do for after-grovel-headers.
Index: src/runtime/Config.x86_64-linux
===================================================================
--- src/runtime/Config.x86_64-linux
+++ src/runtime/Config.x86_64-linux	2012-12-01 21:35:41.518054160 +0100
@@ -43,6 +43,7 @@
 endif
 
 CFLAGS += -fno-omit-frame-pointer
+CFLAGS += $(RPM_OPT_FLAGS)
 
 GC_SRC = gencgc.c
 
Index: contrib/asdf-module.mk
===================================================================
--- contrib/asdf-module.mk
+++ contrib/asdf-module.mk	2012-12-01 21:36:40.433337538 +0100
@@ -25,6 +25,8 @@
   EXTRA_CFLAGS=-D_GNU_SOURCE
 endif
 
+EXTRA_CFLAGS += $(RPM_OPT_FLAGS)
+
 export CC SBCL EXTRA_CFLAGS EXTRA_LDFLAGS
 
 all: $(EXTRA_ALL_TARGETS)
