--- ./src/runtime/linux-os.c.PERS	2013-11-30 16:28:19.000000000 +0200
+++ ./src/runtime/linux-os.c	2014-01-21 18:39:45.465419969 +0200
@@ -288,6 +288,11 @@
                     setenv("SBCL_IS_RESTARTING", "T", 1);
                     runtime[i] = '\0';
                     execv(runtime, argv);
+                } else {
+                    setenv("SBCL_IS_RESTARTING", "T", 1);
+                    /* if /proc isn't available (like in chroot builds, like mock),
+                     * try using execvp with argv[0] instead */
+                    execvp(argv[0], argv);
                 }
             }
             /* Either changing the personality or execve() failed. Either
